beacon-chain/cache/bloom_filter.go

```diff
@@ -0,0 +1,64 @@
+package cache
+
+import (
+	"encoding/binary"
+	"sync"
+
+	"github.com/willf/bloom"
+)
+
+// ActiveIndicesBloomFilter defines BloomFilter for storing and twsting of active indices
+// Bloom Filter is a space-efficient probabilistic data structure
+// that is used to test whether an element is a member of a set
+type ActiveIndicesBloomFilter struct {
+	filter *bloom.BloomFilter
+	lock   sync.RWMutex
+}
+
+// A Bloom filter has two parameters: m - maximum size, k -the number of hashing functions on elements of the set
+const (
+	m = 300000
+	k = 5
+)
+
+// NewBloomFilter creates a new Bloom Filter for storing and testing active indices
```

Comment:
Please add punctuation to the end of all go comments

Reply:
done